week2/The system of the student's score management. v1.0.cpp

```diff
@@ -0,0 +1,172 @@
+#include <stdio.h>
+
+void readscore(long student_ID[], int score[], int n);
+void print_total_and_average(int n, int score[]);
+void rank_by_grade_in_descending_order(long student_ID[], int score[], int n);
+void print_score(long student_ID[], int score[], int n);
+void rank_by_student_ID_in_ascending_order(long student_ID[], int score[], int n);
+int search_student_rank_and_grade_by_student_ID(long student_ID[], int score[], int n);
+void count_the_percentage_of_the_each_category(long student_ID[], int score[], int n);
+
+int main()
+{
+	long student_ID[30];
+	int score[30], n, choice;
+	printf("1.Input record\n");

```

Comment:
Nice to add a space after number label. E.g. "1. Input record". This is more natural for native English user.